scripts/aspnet.benchmarks.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=ASP.NET Benchmarks Server
+
+# Start after network has at least a bound IP address; stop before network is stopped
+After=network.target
+After=network-online.target
```

Comment:
This line does not "pull in" the `network-online.target`. It controls only ordering; `Requires` and `Wants` are completely orthogonal.